### PR TITLE
Time: Update

### PR DESCRIPTION
**Pull request by jaxellis**
With Userge coming out as 1.0.0 I figured it was time to update my old time plugin.

* New flag -code | -c to use a country code manually
* Switch info logs to debug.
* New output style. (Check Below)
* Update original -l flag with -list
* Proper error catching. Should never fail to output.
* Switch list link to hosting on github.
No need to worry about third party providers keeping list.

Old:
It is currently 11:52 PM on 10-03-2022 in America/Los Angeles

New:
It's 11:52PM on Thursday March 10th in America/Los Angeles.

### CODE REVIEW

**Comment by jaxellis** [comment]
@Krishna-Singhal in your commit you removed a return statement which breaks the code. ln97 return False should be the final return statement in that def.

File: plugins/utils/time/__main__.py [hunk 1]
```diff
@@ -84,14 +84,15 @@ async def flag_checks(message: Message, log):
...
             return None
         return country_input
-    elif not COUNTRY_CITY:
+    if not COUNTRY_CITY:
         log.debug("Time: No Config Set")
         await message.edit(default_message, disable_web_page_preview=True,
                            parse_mode="html", del_in=30)
-        return None
-    return False
+    return None
```
Comment (jaxellis) [inline plugins/utils/time/__main__.py:98]: Return None should be for the if not Country_City
Return False should be the final return statement for that def

Comment (rking32) [reply plugins/utils/time/__main__.py:98]: thanks, maybe he didn't see that properly :)

**Comment by rking32** [comment]
btw, thanks for this nice update :)

**Pull request merged**